shared/uavobjectdefinition/actuatorsettings.xml

```diff
@@ -1,12 +1,12 @@
 <xml>
     <object name="ActuatorSettings" singleinstance="true" settings="true">
         <description>Settings for the @ref ActuatorModule that controls the channel assignments for the mixer based on AircraftType</description>
-        <field name="ChannelUpdateFreq" units="Hz" type="uint16" elements="6" defaultvalue="50"/>
-        <field name="ChannelMax" units="us,us/12" type="uint16" elements="10" defaultvalue="2000"/>
-        <field name="ChannelNeutral" units="us,us/12" type="uint16" elements="10" defaultvalue="1000"/>
-        <field name="ChannelMin" units="us,us/12" type="uint16" elements="10" defaultvalue="1000"/>
-        <field name="ChannelType" units="" type="enum" elements="10" options="PWM,OneShot,MK,ASTEC4,PWM Alarm Buzzer,Arming led,Info led" defaultvalue="PWM"/>
-        <field name="ChannelAddr" units="" type="uint8" elements="10" defaultvalue="0,1,2,3,4,5,6,7,8,9"/>
+        <field name="TimerUpdateFreq" units="Hz" type="uint16" elements="6" defaultvalue="50"/>
+        <field name="TimerPwmResolution" units="" type="enum" elements="6" options="1us,80ns" defaultvalue="1us"/>
```

Comment:
I'd leave default at 50 Hz, that's the safe option.  I still have some tricopter's with 50 Hz analog servo's.  I'd rather have to change setup parameters than a burnt out servo FWIW........